docs/romanisim/image-input.rst

```diff
@@ -0,0 +1,52 @@
+Image Input
+===========
+
+The simulator can alternatively take as input a catalog that references a separate list of images of sources.  The concept here is that users may have noiseless images of galaxies from, for example, hydro simulations, and may wish to inject those into simulated images at specific locations.  Or users may have reference images from, for example, the COSMOS field, which could be used as truth images for simulations.
```

Comment:
This section is a bit confusing when it comes to what the total inputs are.. maybe a sentence with a list? I like the mention of the test to show how to work through this.

Reply:
I've changed this to try to more consistently distinguish "input images" from "output images", but I agree that it's confusing.